### PR TITLE
added batch mode to plugin install documentation

### LINKED ISSUE #731: [FEATURE] add missing documentation for batch mode documentation of opensearch-plugin install
### Is your feature request related to a problem?
`opensearch-plugin install` has a `--batch` / `-b` parameter which is currently undocumented.

### What solution would you like?
the parameter must be documented.

### What alternatives have you considered?
as it already existed in ES 7.10.2 their documentation can be used as a reference for now...

### Do you have any additional context?
see the ES documentation on this: https://www.elastic.co/guide/en/elasticsearch/plugins/7.10/_other_command_line_parameters.html#_batch_mode

for further details please check with whoever owns the implementation (fun-fact: with a brief search on GitHub i couldn't find the actual code for this...)

### PR DESCRIPTION
**Pull request by ariamarble**
Signed-off-by: ariamarble <armarble@amazon.com>

### Description
Adds documentation of the --batch option for the installation of plugins

### Issues Resolved
#731 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: _opensearch/install/plugins.md [hunk 1]
```diff
@@ -40,6 +40,15 @@ bin/opensearch-plugin install <name|Zip File|Url>
 
 Restart your OpenSearch node after installing a plugin.
 
+## Batch mode
+
+When installing plugins that require additional privileges not included by default, the plugins will prompt the user for confirmation of the required privileges. The confirmation prompt can be skipped and grant all requested privileges for the plugins being installed by using batch mode.
```
Comment (alicejw-aws) [inline _opensearch/install/plugins.md:45]:
```suggestion
When installing plugins that require additional privileges not included by default, the plugins will prompt the user for confirmation of the required privileges. To grant all requested privileges, use batch mode to skip the confirmation prompt.
```

**Review by alicejw-aws — APPROVED** [review]
LGTM Good job Aria! I only gave a suggestion to simplify and put in active voice.

**Review by cwillum — APPROVED** [review]
LGTM

**Comment by alicejw-aws** [comment]
Hi @ariamarble, the commit from my inline comment suggestion is flagging the DCO signoff error. 
Could you instead of "committing suggested change," just make a separate change and do the regular DCO signoff with your name? 
Sorry i should have mentioned that - i always use the inline comment tool to be precise, but i guess the "commit suggested change" feature creates a garbled DCO signature.
thanks,
Alice

**Pull request merged**